transaction-client/src/main/java/org/apache/omid/tso/client/TSOClient.java

```diff
@@ -342,7 +344,7 @@ public void nodeChanged() throws Exception {
         setTSOAddress(hp.getHost(), hp.getPort());
         epoch = Long.parseLong(currentTSOAndEpochArray[1]);
         LOG.info("CurrentTSO ZNode changed. New TSO Host & Port {}/Epoch {}", hp, getEpoch());
-        if (currentChannel != null && currentChannel.isConnected()) {
+        if (currentChannel != null && currentChannel.isActive()) {
```

Comment:
Might be nice to consolidate this into a helper since you're modifying it.

Reply:
Ah, sorry, I was suggesting to consolidate these two things into one method, aka `boolean isChannelActive(Channel) { return channel != null && channel.isActive();}`

Can ignore if you don't think there's value in it :)